Flabbergasted-Flamingos/bangazon-commerce-app
# Remove customer model and added user as foreign key

# Description
I have taken out all the foreign keys that point to User and am just the django auth user model. Seeder data is still missing, but we'll have to work from hand for now.

The foreign key as user on the other models should enable work to move forward for now.

## Number of Fixes
1. The customer model is commented out until it's needed.

## Related Ticket(s)
#15

## Problem to Solve
The customer model was extending the user and was breaking, but all we needed was the location of customer. we have decided to add location to the product later on if we really need it.

## Proposed Changes
Now when you log in you should only have the auth_user fields availalble. Please let me know if we need more fields in the initial register page.

## Expected Behavior
Register is affected

## Steps to Test Solution

1. 
```
git fetch --all
git checkout rb_landing#15
rm -rf website/migrations
rm db.sqlite3
python manage.py makemigrations website
python manage.py migrate
python manage.py runserver
```
2. Go and register a user, you should be redirected to welcome to bangazon _without_ errors
